lib/validator.js

```diff
@@ -62,11 +44,23 @@ module.exports = class Validator {
      * @returns {ValidationResult}
      */
     validateCommitMessage (commitMessage) {
-        if (this._isSingleLineCommitMessage(commitMessage)) {
-            return this._validateSingleLineCommitMessage(commitMessage);
-        } else {
-            return this._validateMultiLineCommitMessage(commitMessage);
+        // If this is a merge commit, just consider it valid.
+        // Otherwise, run the commit message through all configured rules.
+        if (CommitMessageParser.isMergeCommit(commitMessage)) {
```

Comment:
No I think you're right. I've followed this practice elsewhere but didn't do it here.